pkg/syncer/admissionhandler/validatepvcannotation_test.go

```diff
@@ -37,8 +40,16 @@ var (
 		},
 	}
 
-	pvcAnnotationAdmissionTestInstance *pvcAnnotationAdmissionTest
-	onceForPVCAnnotationAdmissionTest  sync.Once
+	pvcAnnotationAdmissionTestInstance       *pvcAnnotationAdmissionTest
+	pvcHealthAnnotationAdmissionTestInstance *pvcHealthAnnotationAdmissionTest
+	onceForPVCAnnotationAdmissionTest        sync.Once
+	onceForPVCHealthAnnotationAdmissionTest  sync.Once
+)
+
+const (
+	accessible           = "accessible"
```

Comment:
Could you use the const define here https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/f76219a3189a24ec4a89ee8db05a4fe83204f5f9/pkg/csi/service/common/constants.go#L248?

Reply:
Done.